internal/featuregates/features.go

```diff
@@ -10,11 +10,11 @@ import "strings"
 */
 
 const (
-	FeatureGatesConfigMapName      = "osc-feature-gates"
-	LayeredImageDeployment         = "LayeredImageDeployment"
-	LayeredImageDeploymentConfig   = "layeredimagedeployment-config"
-	AdditionalRuntimeClasses       = "AdditionalRuntimeClasses"
-	AdditionalRuntimeClassesConfig = "additionalruntimeclasses-config"
+	FeatureGatesStatusConfigMapName = "osc-feature-gates"
```

Comment:
Again, a good change but should be squashed with the commit where the original name was first introduced.